git/git.go

```diff
@@ -1,40 +1,23 @@
 package git
 
 import (
-	"bufio"
 	"fmt"
 	"os/exec"
+	"strings"
 )
 
 var execCommand = exec.Command
 
 // Tags returns the list of git tags as a string slice
 func Tags() ([]string, error) {
 	cmd := execCommand("git", "tag")
-	outReader, err := cmd.StdoutPipe()
+	out, err := cmd.Output()
```

Comment:
This may also become an issue if you have thousands of tags (something we ran into with thorscmversion)